python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -467,6 +470,11 @@ cdef class ParquetWriter:
     max_page_size_rows: int, default 20000
         Maximum number of rows of each page of the output.
         By default, 20000 will be used.
+    force_nullable_schema : bool, default False.
+        If True, writes all columns as `null` in schema.
+        If False, writes all columns as `not null` in schema,
+        however if a column contains null values, this parameter
+        is ignored.
```

Comment:
I don't like the `If False, writes all columns as `not null` in schema` part.
I'd rather phrase this as "in <this case>, <this> happens, in the <other case> the <other thing> happens"

Reply:
I'm not sure if I understand @vuule's proposal, but this is what I would write:
```suggestion
        If False, columns are written as `null` if they contain null values,
        otherwise as `not null`.
```

Reply:
Sounds good to me 👍